.github/CODEOWNERS

```diff
@@ -1301,6 +1301,7 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 /x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics @elastic/security-detection-engine
 /x-pack/test/security_solution_cypress/cypress/e2e/exceptions @elastic/security-detection-engine
 /x-pack/test/security_solution_cypress/cypress/e2e/overview @elastic/security-detection-engine
+x-pack/test/security_solution_api_integration/test_suites/detections_response/exceptions  @elastic/security-detection-engine
```

Comment:
Can we also map the `rule_creation` and `utils` folders to specific teams?
Let's also assign the `x-pack/test/security_solution_api_integration/test_suites/detections_response` folder to the parent team.